src/object/schedcontext.c

```diff
@@ -278,6 +277,9 @@ void schedContext_bindTCB(sched_context_t *sc, tcb_t *tcb)
 
     SMP_COND_STATEMENT(migrateTCB(tcb, sc->scCore));
 
+    if (sc_active(sc)) {
```

Comment:
This should have `sc_sporadic(tcb->tcbSchedContext)` I think ?

Reply:
I think so. In general, Not guarding any of the `refill_unblock_check` that need to happen for a sporadic SC wont cause issue if the SC isn't sporadic, it just means that they're redundant.